apps/frontend/src/libs/components/date-input/date-input.tsx

```diff
@@ -0,0 +1,109 @@
+import { Calendar } from "react-calendar";
+import {
+	type Control,
+	type FieldPath,
+	type FieldValues,
+} from "react-hook-form";
+
+import { Icon, Popover } from "~/libs/components/components.js";
+import { formatDate } from "~/libs/helpers/helpers.js";
+import {
+	useCallback,
+	useFormController,
+	usePopover,
+} from "~/libs/hooks/hooks.js";
+
+import { addDays, getDifferenceInDays } from "./libs/helpers/helpers.js";
+import { type DateValue } from "./libs/types/types.js";
+import styles from "./styles.module.css";
+
+type Properties<T extends FieldValues> = {
```

Comment:
we need to pass maxDate as today, I meant we don't need to pass minDate for now, because we didn't decide on restriction 😅